projects/storefrontstyles/scss/components/misc/popover/_popover.scss

```diff
@@ -0,0 +1,174 @@
+$cx-popover-font-size: 0.875rem !default;
+$cx-popover-background: var(--cx-color-inverse) !default;
+$cx-popover-max-width: 280px !default;
+
+$cx-popover-border-width: 1px !default;
+$cx-popover-border-color: var(--cx-color-light) !default;
+$cx-popover-border-radius: 5px !default;
+
+$cx-popover-arrow-width: 1rem !default;
+$cx-popover-arrow-height: 0.5rem !default;
+$cx-popover-arrow-color: $cx-popover-background !default;
+
+$cx-popover-z-index: 10 !default;
+$cx-popover-box-shadow-color: rgba(0, 0, 0, 0.1) !default;
+
+%cx-popover {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: $cx-popover-z-index;
+  display: block;
+  max-width: $cx-popover-max-width;
+  font-size: $cx-popover-font-size;
+  word-wrap: break-word;
+  background-color: $cx-popover-background;
+  background-clip: padding-box;
+  border: $cx-popover-border-width solid $cx-popover-border-color;
+  border-radius: $cx-popover-border-radius;
+  padding: 0.5rem;
+  box-shadow: 0px 1px 11px 2px $cx-popover-box-shadow-color;
+  > .arrow {
+    position: absolute;
+    display: block;
+    width: $cx-popover-arrow-width;
+    height: $cx-popover-arrow-height;
+    margin: 0 $cx-popover-border-radius;
+    &::before,
+    &::after {
+      position: absolute;
+      display: block;
+      content: '';
+      border-color: transparent;
+      border-style: solid;
+    }
+  }
+
+  &.top,
+  &.top-left,
+  &.top-right {
+    margin-bottom: $cx-popover-arrow-height;
+    > .arrow {
+      bottom: subtract(-$cx-popover-arrow-height, $cx-popover-border-width);
+      &::before {
+        bottom: 0;
+        border-width: $cx-popover-arrow-height ($cx-popover-arrow-width / 2) 0;
+        border-top-color: $cx-popover-border-color;
+      }
+      &::after {
+        bottom: $cx-popover-border-width;
+        border-width: $cx-popover-arrow-height ($cx-popover-arrow-width / 2) 0;
+        border-top-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.right,
+  &.right-top,
+  &.right-bottom {
+    margin-left: $cx-popover-arrow-height;
+    > .arrow {
+      left: subtract(-$cx-popover-arrow-height, $cx-popover-border-width);
+      width: $cx-popover-arrow-height;
+      height: $cx-popover-arrow-width;
+      margin: $cx-popover-border-radius 0;
+      &::before {
+        left: 0;
+        border-width: ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2) 0;
+        border-right-color: $cx-popover-border-color;
+      }
+      &::after {
+        left: $cx-popover-border-width;
+        border-width: ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2) 0;
+        border-right-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.left,
+  &.left-top,
+  &.left-bottom {
+    margin-right: $cx-popover-arrow-height;
```

Comment:
Any padding and margin-left/right should be changed to -inline-start and -inline-end.
(padding or margin)-inline-start for left
(padding or margin)-inline-end for right

Reply:
Updated